app/src/main/java/com/beemdevelopment/aegis/vault/VaultBackupManager.java

```diff
@@ -51,10 +53,10 @@ public VaultBackupManager(Context context, AuditLogRepository auditLogRepository
         _auditLogRepository = auditLogRepository;
     }
 
-    public void scheduleBackup(File tempFile, Uri dirUri, int versionsToKeep) {
```

Comment:
We should keep passing these settings in instead of reading from Preferences here. There's a potential race where the preferences can change while the backup is running.